dipy/viz/ui.py

```diff
@@ -693,3 +700,523 @@ def re_align(self, window_size_change):
                              self.center[1] + window_size_change[1]))
         else:
             raise ValueError("You can only left-align or right-align objects in a panel.")
+
+
+class TextActor2D(vtkTextActor):
+    """ Inherits from the default vtkTextActor and helps setting the text.
+
+    Contains member functions for text formatting.
+
+    """
+    def message(self, text):
+        """ Set message after initialization.
+
+        Parameters
+        ----------
+        text : string
+            The message to be set.
+
+        """
+        self.SetInput(text)
+
+    def set_message(self, text):
+        """ Modify text message.
+
+        Parameters
+        ----------
+        text : string
+            The new message.
+
+        """
+        self.SetInput(text)
+
+    def get_message(self):
+        """ Gets message from the text.
+
+        Returns
+        -------
+        message : string
+            The current text message.
+
+        """
+        return self.GetInput()
+
+    def font_size(self, size):
+        """ Sets font size.
+
+        Parameters
+        ----------
+        size : int
+            Input font size.
+
+        """
+        self.GetTextProperty().SetFontSize(size)
+
+    def font_family(self, family='Arial'):
+        """ Sets font family.
+
+        Currently defaults to Ariel.
+
+        Parameters
+        ----------
+        family : string
+            The font family.
+
+        """
+        if family == 'Arial':
+            self.GetTextProperty().SetFontFamilyToArial()
+        # ToDo: Add other font families.
+
+    def justification(self, justification):
+        """ Justifies text.
+
+        Parameters
+        ----------
+        justification : string
+            Possible values are left, right, center
+
+        """
+        text_property = self.GetTextProperty()
+        if justification == 'left':
+            text_property.SetJustificationToLeft()
+        if justification == 'center':
+            text_property.SetJustificationToCentered()
+        if justification == 'right':
+            text_property.SetJustificationToRight()
+
+    def font_style(self, bold=False, italic=False, shadow=False):
+        """ Style font.
+
+        Parameters
+        ----------
+        bold : bool
+        italic : bool
+        shadow : bool
+
+        """
+        text_property = self.GetTextProperty()
+        if bold:
+            text_property.BoldOn()
```

Comment:
Yep, there is one: http://www.vtk.org/doc/nightly/html/classvtkTextProperty.html#acc63bcbc7b4fef1b2cc3804d96d2cafe
But I'm not sure what the integral argument in it is. I'll look further.